.github/workflows/package.yml

```diff
@@ -0,0 +1,56 @@
+name: package
+
+on:
+  push:
+    branches:
+      - next
+      - cd_*
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
I think the workflow is only triggered when creating a tag

Reply:
@namhoang1604 , I removed branches but left workflow dispatch option. it is handy when a workflow can be ran manually.